cypress/integration/nameWrapper.spec.js

```diff
@@ -0,0 +1,189 @@
+const ROOT = Cypress.env('ROOT')
+const NAME_ROOT = Cypress.env('NAME_ROOT')
+
+const DISABLED_COLOUR = 'rgb(223, 223, 223)'
+
+function waitUntilInputResolves(buttonTextOrOptions) {
+  return cy.waitUntilInputResolves(buttonTextOrOptions)
+}
+
+function waitUntilTestIdDoesNotExist(testId) {
+  return cy.waitUntilTestIdDoesNotExist(testId)
+}
+
+function waitUntilTextDoesNotExist(text) {
+  return cy.waitUntilTextDoesNotExist(text)
+}
+
+function confirmRecordUpdate() {
+  cy.wait(1000)
+  cy.getByTestId('action').click({ force: true })
+  cy.wait(1000)
+  cy.getByTestId('send-transaction').click({
+    force: true
+  })
+  cy.wait(1000)
+}
+
+function refreshAndCheckText(url, textOrArrayOfText) {
+  cy.visit(url)
+  cy.wait(10000)
+  if (typeof textOrArrayOfText === 'string') {
+    cy.queryByText(textOrArrayOfText, { timeout: 20000 }).should('exist')
+  } else {
+    textOrArrayOfText.forEach(text =>
+      cy.queryByText(text, { timeout: 20000 }).should('exist')
+    )
+  }
+}
+
+describe('Name Wrapper Tests', () => {
```

Comment:
Test descriptions should read more like a sentence describing the behaviour expected:

```
describe('page with wrapped name', () => {
   it('should be in read only mode', () => {})
   it('should display edit name wrapper banner', () => {})
   it('should have subdomains be read only', () => {})
})
```